README.md

```diff
@@ -1,2 +1,14 @@
 # test-infra
-test-infra configurations
+
+test-infra is a CI/CD system aimming at kitex-contrib's Open Source Project Management.
+
+## features
+
+1. GitHub automation in the form of policy enforcement.
+2. chat-ops via /foo style commands.
+3. automatic PR merging.
+
+In this repository, you can configure:
+
+1. Project's Member Management.
```

Comment:
let's change from "Project's Member Management." to "Organization's members."

Reply:
OK